src/main/java/com/zextras/utils/ReflectionUtils.java

```diff
@@ -0,0 +1,52 @@
+package com.zextras.utils;
+
+import com.zextras.transcoders.reflection.DefaultReflectionTranscoder;
+
+import java.lang.reflect.Field;
+
+/**
+ * Helper util class that performs reflecting operations.
+ */
+public class ReflectionUtils {
+
+  /**
+   * Sets a given field with a given value using reflection.
+   *
+   * @param field  a field of an object that you want to modify.
+   * @param object an instance of a class that you want to modify.
+   * @param value  a value that you want to have after set operation.
+   */
+  public static void setField(Field field, Object object, Object value) {
+    field.setAccessible(true);
+    try {
+      field.set(object, value);
+    } catch (IllegalAccessException var) {
```

Comment:
var is a restricted identifier, please change it to something else.

Reply:
done!